components/BlogSection/BlogSection.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+import PostPreview from "components/PostPreview/PostPreview";
+
+interface IBlogSection {
+  title?: string;
+  posts: Array<any>;
+  variant: string;
+  filter?: boolean;
+}
+
+const BlogSection = ({ title, variant, filter, posts }: IBlogSection) => {
+  return (
+    <div>
+      {title}
+      {posts.map((element, key) => (
+        <PostPreview
+          post={element}
+          variant={variant}
+          key={`${element.title}-preview-${key}`}
+        />
+      ))}
+    </div>
+  );
+};
```

Comment:
This is fine for ticket GLB-484. However, let's think carefully as to how to implement the ticket GLB-485 with this component.